cookbook/frontend/bower.rst

```diff
@@ -0,0 +1,125 @@
+.. index::
+    single: Front-end; Bower
+
+Using Bower with Symfony
+========================
+
+Symfony and all its packages are perfectly managed by Composer. Bower is a
+dependency management tool for front-end dependencies, like Twitter Bootstrap
+or jQuery. Since Symfony isn't purely a back-end framework, it can't deliver
```

Comment:
hehe, I started with "Since Symfony isn't a front-end framework" and changed it to back-end, but forgot to remove the negation :)